services/task-svc/migrations/000001_init.down.sql

```diff
@@ -0,0 +1,3 @@
+DROP TABLE IF EXISTS tasks;
+DROP TABLE IF EXISTS patients;
```

Comment:
Not a problem here, but in general it is better to uphold the convention of dropping things in the reverse order they were created, to prevent constraint issues